torch_geometric/distributed/__init__.py

```diff
@@ -1,3 +1,17 @@
+from .rpc import (
+    init_rpc,
+    shutdown_rpc,
+    rpc_is_initialized,
+    global_all_gather,
+    global_barrier,
+    RpcRouter,
+    rpc_partition2workers,
+    RpcCallBase,
+    rpc_register,
+    rpc_request_async,
+    rpc_request_sync,
+)
```

Comment:
Is this all necessary to expose as part of `__init__`? Would be in favor of dropping this.

Reply:
done